dashboard/test/models/user_test.rb

```diff
@@ -2940,6 +2940,115 @@ class RecentCoursesAndScripts < ActiveSupport::TestCase
     end
   end
 
+  test 'from_omniauth: creates new student user without saving provided given_name' do
```

Comment:
New test should be written in [Specs format.](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#writing-unit-tests)
Additionally, I'd like to take a close look and determine which tests are purely unit tests VS. which ones are integration.